students/13th/choihaneul/login.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>repl.it</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+    <link href="style/login.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <div class="login__container">
+      <h1 class="logo"><img src="img/logo_text.png" alt="instagram_logo" /></h1>
```

Comment:
```suggestion
      <h1 class="logo">
          <img src="img/logo_text.png" alt="instagram_logo" />
      </h1>
```
- 태그가 바뀌는 부분은 무조건 줄바꿈 처리 해주세요! 가독성이 좋아집니다.